easybuild/easyconfigs/n/novoalign/novoalign-3.09.00.eb

```diff
@@ -0,0 +1,52 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Modified by Adam Huffman, Jonas Demeulemeester and Kerstin Haase
+# The Francis Crick Institute
+
+easyblock = 'BinariesTarball'
+
+name = 'novoalign'
+version = '3.09.00'
+
+homepage = 'http://www.novocraft.com/products/novoalign/'
+description = """Map short reads onto a reference genome from Illumina,
+ Ion Torrent, and 454 next generation sequencing platforms"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Download manually from http://www.novocraft.com/support/download/
+# source_urls = ['http://www.novocraft.com/support/download/download.php?filename=V(%version)s/']
+sources = ['novocraftV%(version)s.Linux3.10.0.tar.gz']
+
+checksums = ['c9d214c22a8234ded9837d412b30bc94f070191c8a02654b1e0d65a66a0e7f06']
+
+postinstallcmds = ["mv *.pdf .."]
```

Comment:
What's the point of this? Isn't this just moving stuff around in the build dir after the installation? I don't see the point of that...